src/zope/viewlet/tests.py

```diff
@@ -113,10 +113,8 @@ def __init__(self, dict):
         self.__dict = dict
 
     def __getattr__(self, name):
-        try:
-            return self.__dict[name]
-        except KeyError:
-            raise AttributeError(name)
+        return self.__dict[name]
+
```

Comment:
Why not do `self.__dict__ = dict` in the constructor and drop the `__getattr__` altogether?

Reply:
Nice suggestion.